services/provider/client/client.go

```diff
@@ -188,12 +189,15 @@ func (cc *OCSProviderClient) GetStorageClassClaimConfig(ctx context.Context, con
 	return cc.Client.GetStorageClassClaimConfig(apiCtx, req)
 }
 
-func (cc *OCSProviderClient) ReportStatus(ctx context.Context, consumerUUID string) (*pb.ReportStatusResponse, error) {
+func (cc *OCSProviderClient) ReportStatus(ctx context.Context, consumerUUID string, status cs.ConsumerStatus) (*pb.ReportStatusResponse, error) {
 	if cc.Client == nil || cc.clientConn == nil {
 		return nil, fmt.Errorf("Provider client is closed")
 	}
+
 	req := &pb.ReportStatusRequest{
-		StorageConsumerUUID: consumerUUID,
+		StorageConsumerUUID:   consumerUUID,
+		ClientClusterVersion:  status.GetClusterVersion(),
+		ClientOperatorVersion: status.GetOperatorVersion(),
```

Comment:
This copy does not make sense. You should have something like
```suggestion
		ClientStatus:  status,
```

Reply:
- we are construction proto message here and I don't think this is possible unless I'm missing something.
- do you mean the proto message should be a opaque object?

Reply:
done.